redis.conf

```diff
@@ -1578,6 +1578,17 @@ lua-time-limit 5000
 #
 # cluster-allow-reads-when-down no
 
+# This option, when set to yes, allows nodes to serve pubsub local traffic while the
+# the cluster is in a down state, as long as it believes it owns the slots.
+#
+# This is useful if the application would like to use the pubsub feature even if
+# during the cluster global stable state is not OK. If the application is only
+# interacting with a single slot and doesn't need to be wary of outage of other node
+# or slot, this feature should be kept as yes.
```

Comment:
Thought about it some more, I could only think of the following case when you would want this to be set to yes. 
```suggestion
# This is useful if the application would like to use the pubsub feature even when
# the cluster global stable state is not OK. If the application wants to make sure only
# one shard is serving a given channel, this feature should be kept as yes.
```